src/code/z_actor.c

```diff
@@ -1715,15 +1715,15 @@ void func_8002F7A0(PlayState* play, Actor* actor, f32 arg2, s16 arg3, f32 arg4)
 }
 
 /**
- * Play a sound effect at the players's position
+ * Play a sound effect at the players' position
  */
 void Player_PlaySfx(Player* player, u16 sfxId) {
     Audio_PlaySfxGeneral(sfxId, &player->actor.projectedPos, 4, &gSfxDefaultFreqAndVolScale,
                          &gSfxDefaultFreqAndVolScale, &gSfxDefaultReverb);
 }
 
 /**
- * Play a sound effect at the actor's position
+ * Play a sound effect at the actors' position
```

Comment:
it should prob be 's cause its possessive (the actor or player's own position)

Reply:
commas always confuse me PepeHands